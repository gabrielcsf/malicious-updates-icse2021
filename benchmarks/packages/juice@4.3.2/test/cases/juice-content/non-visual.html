<html>
<head>
<style>
        * { color: red; }
    </style>
    <meta name="key" content="value">
    <script></script>
</head>
<body>
    <p></p>
</body>
</html>
