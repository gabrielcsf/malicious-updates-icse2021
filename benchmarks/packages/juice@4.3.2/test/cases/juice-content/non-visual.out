<html style="color: red;">
<head>

    <meta name="key" content="value">
    <script></script>
</head>
<body style="color: red;">
    <p style="color: red;"></p>
</body>
</html>
